wrappers/netcore/EventSender.csproj

```diff
@@ -0,0 +1,27 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+  </PropertyGroup>
+  <ItemGroup>
+    <None Include="sdk-config.json" CopyToOutputDirectory="Always" />
+    <None Include="EventSender.xml">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </None>
+    <!--
+        TODO: this example assumes at the moment that it's being built on Windows.
+        The entire code works on all other platforms as well, but the native libs
+        name could be different. Depending on what native platform library it is -
+        adjust the native libs path accordingly.
+    -->
+    <None Include="$(ProjectDir)\..\..\Solutions\out\Debug\x64\win32-dll\*.*">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </None>
+    <!-- Don't show included native libraries in Visual Studio editor -->
+    <Content Include="$(ProjectDir)\..\..\Solutions\out\Debug\x64\win32-dll\*.*" Visible="False" />
+  </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="System.Json" Version="4.7.0" />
```

Comment:
While this works for .NET Standard 2.x , unfortunately Newtonsoft works better in Unity. I'm not changing it here since this will be taken care of elsewhere, in the implementation of Unity Package outside of 1DS repository.